WooCommerce/Classes/ViewRelated/ReusableViews/TwoColumnSectionHeaderView.swift

```diff
@@ -21,6 +21,7 @@ class TwoColumnSectionHeaderView: UITableViewHeaderFooterView {
         }
         set {
             leftColumn.text = newValue?.uppercased()
+            leftColumn.isHidden = newValue == nil
```

Comment:
nit: maybe we can also check if the `newValue` is empty in case an empty string is set instead of `nil`? 😄